src/types.ts

```diff
@@ -0,0 +1,82 @@
+export type AttributeNames = Record<string, string>;
+export type AttributeValues = Record<string, any>;
+
+interface BaseScanInput {
+  index?: string;
+  limit?: number;
+  startAt?: string;
+  filterExpression?: string;
+  attributeNames?: AttributeNames;
+  attributeValues?: AttributeValues;
+  consistentRead?: boolean;
+}
+
+export interface ScanInput extends BaseScanInput {
+  segment?: number;
+  totalSegments?: number;
+}
+
+export interface CountOutput {
+  count?: number;
+  scannedCount?: number;
+  lastKey?: string;
+}
+
+export interface Types extends BaseScanInput {
```

Comment:
Also, I believe the removal of `QueryInput` is actually a breaking change since it was previously exported and made available to consumers. 

I wonder if we should consider leveraging something like [dts-jest](https://github.com/ikatyang/dts-jest) to help capture breaking type changes.

Reply:
Wow, thank you for catching that!  I am mystified as to why that changed, that should have just been an extraction.  I wonder if something funky happened when I setup the import from `./types`

I will fix that and check to see if there are any other inadvertent changes.